test/functional/feature_block.py

```diff
@@ -824,7 +824,7 @@ def run_test(self):
         tx.vin.append(CTxIn(COutPoint(b64a.vtx[1].sha256, 0)))
         b64a = self.update_block("64a", [tx])
         assert_equal(len(b64a.serialize()), MAX_BLOCK_BASE_SIZE + 8)
-        self.sync_blocks([b64a], success=False, reject_code=1, reject_reason=b'error parsing message')
+        self.sync_blocks([b64a], success=False, reject_reason='non-canonical ReadCompactSize(): iostream error')
```

Comment:
This is `non-canonical ReadCompactSize(): iostream stream error` on Windows. See https://ci.appveyor.com/project/DrahtBot/bitcoin/build/master.972. Is there a way to specify the second variant?

Reply:
FYI
On macOS, this is ```'non-canonical ReadCompactSize(): unspecified iostream_category error'```.
I think #14007 will also fix this because of discarding the message to be ```'non-canonical ReadCompactSize()```.